main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -563,6 +588,28 @@ object Completion {
     */
   case class VarCompletion(name: String) extends Completion
 
+  /**
+   * Represents a Declaration completion
+   *
+   * @param name the name of the declaration to complete.
+   */
+  case class DeclarationCompletion(name: String) extends Completion
+
+  /**
+   * Represents a Java Class completion
+   *
+   * @param name the name of the java class to complete.
+   */
+  case class JavaClassCompletion(name: String) extends Completion
+
+  /**
+   * Represents a local def completion
+   *
+   * @param sym the symbol of the local function
+   * @param fparams the formal parameters of the local function
+   */
+  case class LocalDefCompletion(sym: Symbol.VarSym, fparams: List[ResolvedAst.FormalParam]) extends Completion
```

Comment:
Do we need to duplicate these? Why can we not reuse what we have?

Reply:
If we really need to above. Perhaps we should call them `LocalDeclCompletion`, `LocalJavaClassCompletion`, and so on-- as you started to do.